src/renderer/actions/settings.js

```diff
@@ -88,3 +88,9 @@ export const dismissBanner = (bannerKey: string) => ({
 
 export const setPreferredDeviceModel = (preferredDeviceModel: DeviceModelId) =>
   saveSettings({ preferredDeviceModel });
+
+// TODO VAL: should I use saveSettings here ????
```

Comment:
i prefer to have explicit action because more reducer could listen to them so all good here.